javalib/src/main/scala/java/io/File.scala

```diff
@@ -299,7 +300,7 @@ class File(_path: String) extends Serializable with Comparable[File] {
       fromCWideString(resolvedName, StandardCharsets.UTF_16LE)
     } else {
       val resolvedName: Ptr[Byte] = alloc[Byte](limits.PATH_MAX)
-      if (realpath(toCString(path), resolvedName) == null) {
+      if (posix.stdlib.realpath(toCString(path), resolvedName) == null) {
```

Comment:
```suggestion
      if (pstdlib.realpath(toCString(path), resolvedName) == null) {
```

Or you might not need the prefix if you moved `realpath`.

Reply:
Eric,

Thank you for the timely review & suggestions.

You encouraged me to do a simpler fix in File.scala, based on the work that you and Wojchiech
had done to layer many files, including stdlib, in posixlib over the ISO C files in clib.  Might as
well benefit from that work.

Anybody in File.scala to begin with is in a world of hurtin'. They need all the simplicity & 
transparency they can get.